vdb-bench-assembly/plugins/vdb-bench-teiid/config.js

```diff
@@ -45,5 +45,5 @@
                    .build());
     }
 
-    hawtioPluginLoader.addModule(pluginName);
+    //hawtioPluginLoader.addModule(pluginName);
```

Comment:
Definitely don't want to disable this since I am intending to add the querying preview button to this module, since we query vdbs rather than data services.